packages/drivers/odsp-driver-definitions/src/odspCache.ts

```diff
@@ -127,5 +127,26 @@ export interface IPersistedCache {
  * @internal
  */
 export function getKeyForCacheEntry(entry: ICacheEntry): string {
-	return `${entry.file.docId}_${entry.type}_${entry.key}`;
+	switch (entry.type) {
```

Comment:
are both these functions used? they are both internal, so we should unify on one, probably this one, since the logic is odsp specific

Reply:
we should also add unit test for the function, which should be pretty straight forward